README.md

```diff
@@ -1,57 +1,190 @@
+    POSTGRES_SERVER_NAME='choose a globally-unique name for your server, e.g. my-postgres-server'
+    POSTGRES_ADMIN_USER='choose an admin user name, e.g. administrator'
+    POSTGRES_ADMIN_PASSWORD='choose a password'
+    APP_DB_NAME='db name for your app, e.g. my-django-db'
+   ```
+
+1. Now export all of the values in this `my-azure-settings` file to your shell. For example, `export VARIABLE = 'value'` in Linux/MacOS or `$env:VARIABLE = 'value'` in powershell. To quickly export all values that are not commented out in bulk in Linux/MacOS, use the command `export $(grep -v '^#' my-azure-settings | xargs)`
+1. From the terminal, run the `create-db.py` file with `python deployment/create-db.py`. Answer all of the questions in the affirmative if it is your first run.
+1. In the last step, you'll be presented with a summary of the postgres DB access details. Make a note of the `fullyQualifiedDomainName`. Copy and paste it into your `my-azure-settings` file under the value `POSTGRES_HOST`.
+1. export this value to your shell  (`export POSTGRES_HOST = 'value that was copied'`) or run the bulk export command from the previous step.
+1. From the terminal, run the `set-deployed-env.py` file to set required environment variables (copied from your shell) to the app service.
+1. Disable App Service's default authentication:
+
+    Navigate to the **Azure App Service** Portal and locate your project. Once you do, click on the **Authentication/Authorization** blade. There, make sure that the **App Services Authentication** is switched off (and nothing else is checked), as this sample is using MSAL for authentication.
+
+    ![disable_easy_auth](./ReadmeFiles/disable_easy_auth.png)
+
+### Step 3: Update your Azure AD App Registration
+
+- Navigate to the home page of your deployed app; take note of and copy the **redirect_uri** displayed on the home page.
+- Navigate back to to the [Azure Portal](https://portal.azure.com).
+- In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations**.
+- In the resulting screen, select the name of your application.
+- In the Authentication blade, paste the URI you copied earlier from your deployed app instance. If the app had multiple redirect URIs, make sure to add new corresponding entries using the App service's full domain in lieu of `localhost:8000` for each redirect URI. Save the configuration.
+- From the *Branding* menu, update the **Home page URL**, to the address of your service, for example `https://example-domain.azurewebsites.net/`. Save the configuration.
+- You're done! Try navigating to the hosted app!
+
+### Step 4: Complete your app deployment settings
+
+Modify your app's `Sample/azure.py` file's allowed hosts as follows, using the full domain name of your app that you recorded in [Step 2: Prepare the app service and database](#step-2-prepare-the-app-service-and-database)
+
+```Python
+ALLOWED_HOSTS = ['https://example-domain.azurewebsites.net']
```

Comment:
shouldn't this (ALLOWED_HOSTS part) go under Step 3? Will we be able to access the hosted app without configuring this?